R/civis_ml.R

```diff
@@ -71,6 +71,10 @@
 #'   data for validation, and \code{skip}, which skips the validation step.
 #' @param verbose Optional, If \code{TRUE}, supply debug outputs in Platform
 #'   logs and make prediction child jobs visible.
+#' @param dvs_to_predict Optional, For scoring, this should be a vector of column
+#'   names of dependent variables to include in the output table. It must be a
+#'   subset of the \code{dependent_variable} vector provided for training.
```

Comment:
Just like the Python client docs, I think it's good to explain that this saves time and disk, and that all scores are calculated either way (so that the model output is not affected, just the user output).